developer_manual/general/codingguidelines.rst

```diff
@@ -11,10 +11,61 @@ General
 * Fixes go directly to master, nevertheless they need to be tested thoroughly
 * New features are always developed in a branch and only merged to master once they are fully done
 * Software should work. Only put features into master when they are complete. It's better to not have a feature instead of having one that works poorly.
-* When you are finished, use the merge request function on Github. The other developers will look at it and give you feedback. You can also post your merge request to the mailing list to let people know. See `the code review page for more information <../bugtracker/codereviews.html>`_
+* When you are finished, use the merge request function on Github to create a pull request. The other developers will look at it and give you feedback. You can signify that your PR is ready for review by adding the label "5 - ready for review" to it. You can also post your merge request to the mailing list to let people know. See `the code review page for more information <../bugtracker/codereviews.html>`_
 * Decisions are made by consensus. We strive for making the best technical decisions and as nobody can know everything, we collaborate. That means a first negative comment might not be the final word, neither is positive feedback an immediate GO. ownCloud is built out of modular pieces (apps) and maintainers have a strong influence. In case of disagreement we consult other seasoned contributors.
 * We need a signed contributor agreement from you to commit into the core repository (apps don't need that). All the information is in our `Contributor agreement FAQ <http://owncloud.org/contribute/agreement/>`_.
 
+Labels
+------
+We assign labels to issues and pull requests to make it easy to find them and to signal what needs to be done. The most important labels and their meaning:
+* #bug - this issue is a bug
+* #enhancement - this issue is a feature request/idea for improvement of ownCloud
+* #design - this needs help from the design team or is a design-related issue/pull request
+* #sharing - this issue or PR is related to sharing
+* #technical debt - this issue or PR is about `technical debt <http://en.wikipedia.org/wiki/Technical_debt>`_
+* #sev1-critical #sev2-high #sev3-medium #sev4-low signify how important the bug is. Data loss gets sev1, XXXX
+* #Junior Job - these are issues which are relatively easy to solve and ideal for people who want to learn how to code in ownCloud
+* Tags showing the state of the issue or PR, numbered 1-6:
+ * #1 - Backlog - this issue has to be tackled but isn't planned in yet
```

Comment:
I'd vote for the backlog milestone - the backlog label is used by hunboard .. no idea if anybody is still using this @butonic